mantis-runtime-loader/src/main/java/io/mantisrx/runtime/loader/RuntimeTask.java

```diff
@@ -30,7 +31,8 @@ public interface RuntimeTask extends Service {
     void initialize(
         String executeStageRequestString,
         String workerConfigurationString,
-        UserCodeClassLoader userCodeClassLoader);
+        UserCodeClassLoader userCodeClassLoader,
+        FailoverStatusClient failoverStatusClient);
```

Comment:
It might be helpful to check if we can avoid leaking autoscaling-specific changes to the runtime task interface. For example, why can it not be initialized within nfmantis-runtime?